src/zope/securitypolicy/securitymap.py

```diff
@@ -146,7 +148,7 @@ def _changed(self):
         if isinstance(map, PersistentSecurityMap):
             map._p_changed = 1
         else:
-            map = PersistentSecurityMap()
+            map = self.map = PersistentSecurityMap()
```

Comment:
This also looks like a bugfix that could use a changelog entry?

Reply:
I viewed it as purely an internal performance fix, which only comes into play the first transaction this object is used in (i.e., rare). But I can add a changelog; if I do that I need to add a test that verifies this particular scenario.

Reply:
Test and note added.